packages/web/components/trade-clipboard/index.tsx

```diff
@@ -379,6 +382,16 @@ export const TradeClipboard: FunctionComponent<{
                     );
                   }}
                   onFocus={() => slippageConfig.setIsManualSlippage(true)}
+                  onClick={() =>
+                    logEvent([
+                      EventName.Swap.inputBoxClicked,
```

Comment:
IMO instead of a click event, we should be using a focus event as we did with matomo.

As though the user intends to add input. Also, this wouldn't trigger if a user tabs into the input box.

Please remove changes to input-box as well.

Reply:
Now auto focus is enabled on input box so I think we don't know what user really did focus. So I think It would be better to track input entered.